x-pack/plugins/reporting/public/panel_actions/get_csv_panel_action.tsx

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { i18n } from '@kbn/i18n';
+
+import { SearchEmbeddable } from 'src/legacy/core_plugins/kibana/public/discover/embeddable/search_embeddable';
```

Comment:
Hm, I'm getting a `Module not found` error with this

Reply:
I've seen that a few times as well, then suddenly it'll work. I'll see what I can find.